addons/binding/org.openhab.binding.network/src/main/java/org/openhab/binding/network/handler/NetworkHandler.java

```diff
@@ -50,20 +52,13 @@ public void handleCommand(ChannelUID channelUID, Command command) {
         if (command instanceof RefreshType) {
             switch (channelUID.getId()) {
                 case CHANNEL_ONLINE:
-                    try {
-                        State state = networkService.updateDeviceState() < 0 ? OnOffType.OFF : OnOffType.ON;
-                        updateState(CHANNEL_ONLINE, state);
-                    } catch (InvalidConfigurationException invalidConfigurationException) {
-                        updateStatus(ThingStatus.OFFLINE);
-                    }
+                    // The GUI requested a refresh. We send the cached value immediately and reset the refresh timer to
+                    // get the device reachable state as fast as possible.
```

Comment:
Do you really think that messing with the scheduled job is the right approach?
You haven't yet considered my https://github.com/openhab/openhab2-addons/pull/1989#issuecomment-286066440.
Note that we have now a [new ExpiringCache service](https://github.com/eclipse/smarthome/pull/3572) in place - wouldn't that be the simplest way to address the issue now?